chainer_pytorch_migration/ignite/__init__.py

```diff
@@ -1,2 +1,13 @@
+import ignite
+
 from .extensions import add_trainer_extension, load_chainer_snapshot  # NOQA
 from .collate import collate_to_array  # NOQA
+
+
+def _get_version(version):
+    return int(version.split('.')[:2])
```

Comment:
I think you should leave a comment that it intentionally compares only up to the minor version.
Otherwise someone could rewrite 2 to 3.